README.md

```diff
@@ -169,6 +169,63 @@ Other strings can be got in following steps:
 
     Add the strings to `boox_strings` in [DeBooxUpx.py](DeBooxUpx.py), use this script to decrypt a `update.upx` file (it can be got by [the method in #2](https://github.com/Hagb/decryptBooxUpdateUpx/issues/2#issuecomment-704006389)), and see if the `update.zip` file is a vaild zip file.
 
+### How to get the strings (Method 2)
+
+In Boox OS since Android 11, strings are moved to `libota_jni.so`, **Method 2** is for this situation. 
+
+`MODEL` string is the model name, exactly the output of `getprop ro.product.model` and/or value of `ro.product.model` in the file `/system/build.prop`.
+
+Other strings can be got in following steps:
+
+1. Get `/system/lib/libota_jni.so` (for 32-bit device) or `/system/lib64/libota_jni.so` (for 64-bit device)
+
+    Connect to device via `adb` as above. Then execute command:
+
+    ```
+    adb pull /system/lib/libota_jni.so
+    ```
+
+    or
+
+    ```
+    adb pull /system/lib/libota_jni.so
```

Comment:
This is the same command as the first one, I think it's supposed to be lib or lib64

Reply:
It's a typo. Fixed.